shared/cfg/guest-hw.cfg

```diff
@@ -352,6 +357,10 @@ variants image_backend:
         nfs_mount_src = localhost:/mnt/nfs_export
         setup_local_nfs = yes
         export_options = rw
+    - nvme_direct:
+        storage_type = nvme
+        enable_nvme = yes
+        remove_image = no
```

Comment:
Better to document the address and namespace param as well.

Reply:
Thanks, have updated.